eventfiles/models.py

```diff
@@ -17,8 +17,17 @@ def validate_unique_file(up_file, up_participant_id):
         if EventFile.objects.filter(file=up_file, participant_id=up_participant_id):
             raise ValidationError('that file already exist in meeting')
 
+    @staticmethod
+    def get_files_by_event(event_id):
+        participants = EventParticipant.objects.filter(event_id=event_id)
+        files = []
+        for participant in participants:
+            files.extend(EventFile.objects.filter(participant_id=participant))
```

Comment:
Can you make line 25 into a command inside a manager of event file.
like: get event_file by participant

the same for line 22 if possible 😄

Reply:
Done